src/EditorFeatures/Test2/IntelliSense/CSharpCompletionCommandHandlerTests.vb

```diff
@@ -1950,5 +1950,33 @@ class Program
                 Assert.Contains("args:", state.GetLineTextFromCaretPosition())
             End Using
         End Function
+
+        <WorkItem(13481, "https://github.com/dotnet/roslyn/issues/13481")>
+        <WpfFact, Trait(Traits.Feature, Traits.Features.Completion)>
+        Public Async Function TestBackspaceSelection1() As Task
+            Using state = TestState.CreateCSharpTestState(
+                <Document><![CDATA[
+using System;
+
+class Program
+{
+    static void Main()
+    {
+        DateTimeOffset$$
```

Comment:
From the bug:

> Note that if the final dot is not typed at step 2., it behaves as expected.

Your test doesn't seem to cover the same scenario as in the bug.